std/random.d

```diff
@@ -415,8 +415,8 @@ $(D x0).
         return _x;
     }
 
-///
-    @property typeof(this) save() @safe pure nothrow
```

Comment:
At first I started also adding `@nogc`, in addition to the other attributes, but then I thought that I should leave it to attribute inference, because the `UintType` could be a BigInteger with non-`@nogc` non-`nothrow`, etc. postblit)